__tests__/extension-requests/extension-requests.test.js

```diff
@@ -563,4 +563,32 @@ describe('Tests the Extension Requests Screen', () => {
 
     expect(isTooltipVisible).toBe(true);
   });
+
+  test('Checks the Request Number and request value element on Extension requests listing page', async () => {
+    const url = 'http://localhost:8000/extension-requests/?dev=true'; // Include the dev parameter in the URL
+    await page.goto(url);
+
+    title = await page.$('.header h1');
+    searchBar = await page.$('#search');
+    filterButton = await page.$('#filter-button');
+    extensionCardsList = await page.$$('.extension-card');
+    extensionRequestsElement = await page.$('.extension-requests');
+
+    expect(title).toBeTruthy();
+    expect(searchBar).toBeTruthy();
+    expect(filterButton).toBeTruthy();
+    expect(extensionCardsList.length).toBe(4);
```

Comment:
why are we verifying all the elements in the page when the test is to check only the request number and value?

Reply:
Removed